Formula/naabu.rb

```diff
@@ -1,9 +1,9 @@
 class Naabu < Formula
   desc "Fast port scanner"
   homepage "https://github.com/projectdiscovery/naabu"
-  url "https://github.com/projectdiscovery/naabu/archive/v1.1.4.tar.gz"
-  sha256 "fb5fae5ef7ef45217a238b59ed16095960c197c6f8b2f460873a445dbc80c2b4"
-  license "GPL-3.0"
```

Comment:
if we are not sure, we can just leave it as it is, I think.

generally, I think folks would still prefer `GPL-3.0-or-later` as compatibility choice, but i could be wrong though. :)

Reply:
I agree that `GPL-3.0-or-later` would be more compatible, but `GPL-3.0-only` is the more conservative choice here. Open to other opinions on how to proceed here.

Reply:
> generally, I think folks would still prefer GPL-3.0-or-later as compatibility choice, but i could be wrong though. :)

We're trying to represent the upstream license here though. Preference has no part in that. 

I'd just keep the identifier as it is and request clarification from upstream

Reply:
agree, that is what i was thinking.